$NetBSD: patch-ai,v 1.3 2006/01/04 20:35:08 joerg Exp $

errno is a macro in a modern libc, and may not be a plain int at all.
The old `extern int errno' declaration collides with it.

--- src/floatfns.c.orig	1995-02-17 07:47:39.000000000 +0000
+++ src/floatfns.c
@@ -49,7 +49,7 @@ the Free Software Foundation, 675 Mass A
 #include "lisp.h"
 #include "syssignal.h"
 
-Lisp_Object Qarith_error;
+extern Lisp_Object Qarith_error;
 
 #ifdef LISP_FLOAT_TYPE
 
@@ -103,8 +103,6 @@ extern double logb ();
 
 #ifdef FLOAT_CHECK_ERRNO
 # include <errno.h>
-
-extern int errno;
 #endif
 
 /* Avoid traps on VMS from sinh and cosh.
